src/kern/arm/bsp/s32z/Modules

```diff
@@ -2,8 +2,10 @@
 
 OBJECTS_LIBUART       += uart_dcc-v6.o
 CXXFLAGS_uart-libuart += $(call LIBUART_UART, dcc-v6, dcc_v6)
-PREPROCESS_PARTS      += arm_generic_timer pic_gic
+PREPROCESS_PARTS      += arm_generic_timer pic_gic mru
```

Comment:
I think this is not needed. Adding `mru` to `INTERFACES_DRIVERS` should be enough.